Installing via Brew fails on macOS Monteray

I try to install the library via `brew` using command `brew install snaipe/soft/criterion` 

But installing fails with error.  Please see full output below

```bash
kotkovetsiv@macbook Subarray Division $ brew install snaipe/soft/criterion
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> Updated Formulae
Updated 2 formulae.
==> Updated Casks
Updated 1 cask.

==> Downloading https://ghcr.io/v2/homebrew/core/nanomsg/manifests/1.1.5
Already downloaded: /Users/kotkovetsiv/Library/Caches/Homebrew/downloads/b9e922ac74ec517db78d67ef92a74caeb60292891d01eab582e4b896a0a721c3--nanomsg-1.1.5.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/nanomsg/blobs/sha256:3f59f226fe111172467d9d7d187aa486659e5b58abc3571b6612b544d4dc43e9
Already downloaded: /Users/kotkovetsiv/Library/Caches/Homebrew/downloads/a89646d0aff533b6758203cc05097506af95dde0ed53b238e393fc39fb57d6ff--nanomsg--1.1.5.monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/cmake/manifests/3.21.4
Already downloaded: /Users/kotkovetsiv/Library/Caches/Homebrew/downloads/b7ef8d6eb909e967d072212c62e71bfb8e94e6227ae2d3567bbabcc561fd9fff--cmake-3.21.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/cmake/blobs/sha256:d6fb0df5818454f9f9074552eb06c34146da91c5212c16057b544a3677a29d9a
Already downloaded: /Users/kotkovetsiv/Library/Caches/Homebrew/downloads/aa67f42e613f2d2295dd9d2722cad83179b924e4877cbbb3f1aff5b1d470a7e5--cmake--3.21.4.monterey.bottle.tar.gz
==> Cloning https://github.com/Snaipe/Criterion.git
Updating /Users/kotkovetsiv/Library/Caches/Homebrew/criterion--git
==> Checking out tag v2.3.3
HEAD is now at a64b860 v2.3.3: merging fix/247.
HEAD is now at a64b860 v2.3.3: merging fix/247.
Entering 'dependencies/debugbreak'
Entering 'dependencies/klib'
Entering 'dependencies/nanopb'
/Users/kotkovetsiv/Library/Caches/Homebrew/criterion--git/dependencies/debugbreak
/Users/kotkovetsiv/Library/Caches/Homebrew/criterion--git/dependencies/klib
/Users/kotkovetsiv/Library/Caches/Homebrew/criterion--git/dependencies/nanopb
==> Installing criterion from snaipe/soft
==> Installing dependencies for snaipe/soft/criterion: nanomsg and cmake
==> Installing snaipe/soft/criterion dependency: nanomsg
==> Pouring nanomsg--1.1.5.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/nanomsg/1.1.5: 25 files, 348.8KB
==> Installing snaipe/soft/criterion dependency: cmake
==> Pouring cmake--3.21.4.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/cmake/3.21.4: 2,953 files, 42.6MB
==> Installing snaipe/soft/criterion
==> cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DI18N=OFF .
==> make install
Last 15 lines from /Users/kotkovetsiv/Library/Logs/Homebrew/criterion/02.make:
[ 86%] Building C object CMakeFiles/criterion.dir/src/protocol/protocol.c.o
[ 88%] Building C object CMakeFiles/criterion.dir/src/protocol/messages.c.o
[ 89%] Building C object CMakeFiles/criterion.dir/src/protocol/connect.c.o
[ 91%] Building C object CMakeFiles/criterion.dir/src/string/extglobmatch.c.o
[ 92%] Building C object CMakeFiles/criterion.dir/src/string/brz.c.o
[ 94%] Building C object CMakeFiles/criterion.dir/dependencies/nanopb/pb_common.c.o
[ 95%] Building C object CMakeFiles/criterion.dir/dependencies/nanopb/pb_encode.c.o
[ 97%] Building C object CMakeFiles/criterion.dir/dependencies/nanopb/pb_decode.c.o
[ 98%] Building C object CMakeFiles/criterion.dir/src/core/theories.c.o
[100%] Linking C shared library libcriterion.dylib
ld: unaligned pointer(s) for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [libcriterion.3.1.0.dylib] Error 1
make[1]: *** [CMakeFiles/criterion.dir/all] Error 2
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/snaipe/homebrew-soft/issues

These open issues may also help:
linker error macOS Mojave 10.14.1 installing Criterion https://github.com/Snaipe/homebrew-soft/issues/3
```

Does anybody know how to fix the issue? Thank you in advance!

--- a/README.md
+++ b/README.md
@@ -1,9 +1,9 @@
 <h1><img src="doc/criterion-title.png" height="96" alt="Criterion Logo" /></h1>
 
-[![Build Status](https://api.cirrus-ci.com/github/Snaipe/Criterion.svg)](https://cirrus-ci.com/github/Snaipe/Criterion) 
-[![Coverage Status](https://img.shields.io/codecov/c/github/Snaipe/Criterion/bleeding.svg)](https://codecov.io/github/Snaipe/Criterion?branch=bleeding) 
-[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Snaipe/Criterion/blob/master/LICENSE) 
-[![Version](https://img.shields.io/github/release/Snaipe/Criterion.svg?label=version)](https://github.com/Snaipe/Criterion/releases/latest) 
+[![Build Status](https://api.cirrus-ci.com/github/Snaipe/Criterion.svg)](https://cirrus-ci.com/github/Snaipe/Criterion)
+[![Coverage Status](https://img.shields.io/codecov/c/github/Snaipe/Criterion/bleeding.svg)](https://codecov.io/github/Snaipe/Criterion?branch=bleeding)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Snaipe/Criterion/blob/master/LICENSE)
+[![Version](https://img.shields.io/github/release/Snaipe/Criterion.svg?label=version)](https://github.com/Snaipe/Criterion/releases/latest)
 
 A dead-simple, yet extensible, C and C++ unit testing framework.
 

--- a/README.md
+++ b/README.md
@@ -46,7 +46,7 @@ the user would have with other frameworks:
     $ sudo apt-get install criterion-dev
     ```
 * Arch Linux ([AUR](https://aur.archlinux.org/packages/criterion/)): `pacaur -S criterion`
-* macOS: `brew install snaipe/soft/criterion`
+* macOS: `brew install mranno/tap/criterion`
 
 ### Binary archives
 

--- a/README.md
+++ b/README.md
@@ -32,7 +32,7 @@ the user would have with other frameworks:
 * [x] Supports parameterized tests and theories.
 * [x] Progress and statistics can be followed in real time with report hooks.
 * [x] TAP output format can be enabled with an option.
-* [x] Runs on Linux, FreeBSD, Mac OS X, and Windows (Compiling with MinGW GCC and Visual Studio 2015+).
+* [x] Runs on Linux, FreeBSD, macOS, and Windows (Compiling with MinGW GCC and Visual Studio 2015+).
 
 ## Downloads
 

--- a/README.md
+++ b/README.md
@@ -91,7 +91,7 @@ Criterion is built with the following projects:
 
 * [boxfort](https://github.com/diacritic/BoxFort)
 * [debugbreak](https://github.com/scottt/debugbreak)
-* [dyncall](http://www.dyncall.org/)
+* [libffi](https://sourceware.org/libffi/)
 * [klib](http://attractivechaos.github.io/klib/)
 * [libcsptr](https://github.com/Snaipe/libcsptr)
 * [nanomsg](http://nanomsg.org/)

--- a/README.md
+++ b/README.md
@@ -38,15 +38,15 @@ the user would have with other frameworks:
 
 ### Packages
 
-* Mac OS X: `brew install snaipe/soft/criterion`
-* [AUR](https://aur.archlinux.org/packages/criterion/): `pacaur -S criterion`
-* Ubuntu/Debian:
+* Ubuntu (>=21.04) / Debian (>=11): `apt-get install libcriterion-dev`, on older versions:
 
     ```bash
     $ sudo add-apt-repository ppa:snaipewastaken/ppa
     $ sudo apt-get update
     $ sudo apt-get install criterion-dev
     ```
+* Arch Linux ([AUR](https://aur.archlinux.org/packages/criterion/)): `pacaur -S criterion`
+* macOS: `brew install snaipe/soft/criterion`
 
 ### Binary archives
 

--- a/doc/intro.rst
+++ b/doc/intro.rst
@@ -30,4 +30,4 @@ Features
 * Supports parameterized tests and theories.
 * Progress and statistics can be followed in real time with report hooks.
 * TAP output format can be enabled with an option.
-* Runs on Linux, FreeBSD, Mac OS X, and Windows (Compiling with MinGW GCC and Visual Studio 2015+).
+* Runs on Linux, FreeBSD, macOS, and Windows (Compiling with MinGW GCC and Visual Studio 2015+).

--- a/doc/setup.rst
+++ b/doc/setup.rst
@@ -15,7 +15,19 @@ The following compilers are supported to compile both the library and the tests:
 Building from source
 --------------------
 
-First, clone this repository:
+First, install dependencies:
+
+* C/C++ compiler
+* Meson, Ninja
+* CMake (for subprojects)
+* pkg-config
+* libffi (libffi-dev)
+* libgit2 (libgit2-dev)
+
+Other runtime dependencies will be bundled if they are not available on the
+system (BoxFort, debugbreak, klib, nanomsg, nanopb).
+
+Clone this repository:
 
 .. code-block:: bash
 

--- a/doc/setup.rst
+++ b/doc/setup.rst
@@ -4,7 +4,7 @@ Setup
 Prerequisites
 -------------
 
-The library is supported on Linux, OS X, FreeBSD, and Windows.
+The library is supported on Linux, macOS, FreeBSD, and Windows.
 
 The following compilers are supported to compile both the library and the tests:
 
@@ -28,7 +28,7 @@ Then, run the following commands to build Criterion:
     $ meson build
     $ ninja -C build
 
-Installing the library and language files (Linux, OS X, FreeBSD)
+Installing the library and language files (Linux, macOS, FreeBSD)
 ----------------------------------------------------------------
 
 Run with an elevated shell:

